src/unity/python/turicreate/visualization/__init__.py

```diff
@@ -3,6 +3,121 @@
 #
 # Use of this source code is governed by a BSD-3-clause license that can
 # be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+#
+# Turi Create Visualization allows you to create Plot objects and 
+# manipulate them using various functionalities, including
+# saving them (.save) and viewing them (.show). Moreover, Turi Create 
+# Visualization also allows the user to call `show` on data sources like 
+# SFrames and SArrays, and provides functionality to create independent plots 
+# like scatter plots, categorical heatmaps, box plots, heatmaps, columnwise 
+# summaries, and item frequency plots.
+#
+# ==========
+# class Plot
+# ==========
+#
+# 	An immutable object representation of a visualization.
+#
+#   Notes
+#   -----
+#   - A plot object is returned via the SFrame's or SArray's .plot() method
+#
+#   Examples
+#   --------
+#   Suppose 'plt' is an Plot Object
+#
+#   We can view it using:
+#   >>> plt.show()
+#
+#   We can save it using:
+#   >>> plt.save('vega_spec.json')
+#
+#   We can also save the vega representation of the plot without data:
+#   >>> plt.save('vega_spec.json', False)
+#
+#   We can save the plot as a PNG/SVG using:
+#   >>> plt.save('test.png')
+#   >>> plt.save('test.svg')
+#
+# ====
+# show
+# ====
+# 
+# 	A method for displaying the Plot object. The plot will render either inline
+# 	in a Jupyter Notebook, or in a native GUI window, depending on the value 
+# 	provided in `turicreate.visualization.set_target` (defaults to 'auto').
+# 
+# ====
+# save
+# ====
+#
+# 	A method for saving the Plot object in a vega representation
+#
+#   Parameters
+#	----------
```

Comment:
This won't render correctly in Sphinx. Can you check how this will render in bumpy docs.